### DISCUSSION TITLE
Glossary - Call for competition

### DISCUSSION THREAD

**Discussion by muricna** [eprocurementontology/eprocurementontology#490] [Ideas]
Term:  Call for competition
Definition: A public announcement of the launch of a procurement procedure.

Additional information:  According to EU Directives this should be via a prior information notice or a contract notice.

WG approval: 14/05/2018

**Comment by JachymHercher** [comment]
I'm not sure "launch of a procurement procedure" is ideal. For example, OCDS (as well as eForms) track the procedure already since the planning phase, i.e. the PIN. From that point of view, the procedure starts before a call for competition.

What about a more functional definition, something along the lines of "A request for tenders or requests to participate to be submitted by economic operators"?

**Comment by eprocurementontology** [comment]
WG approval 14/05/2018

Term: Call For Competition
Definition: A public announcement of the launch of a procurement procedure.

Additional information/WG discussion points: According to EU Directives this should be via a prior information notice or a contract notice.

The glossary and the conceptual model have been updated accordingly on 29/06/2018.

**Comment by muricna** [comment]
To be reviewed by working group in view of Jáchym's comment
